interview_db/views.py

```diff
@@ -1,62 +1,125 @@
 # Copyright 2022 UW-IT, University of Washington
 # SPDX-License-Identifier: Apache-2.0
 
-
 from django.conf import settings
-from django.shortcuts import render, get_object_or_404
-from django.utils.decorators import method_decorator
-from django.views.generic.list import ListView
-
-from uw_saml.decorators import group_required
+from django.views.generic import TemplateView
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from django.db.models import Q
 
-from .models import Code, Interview
+from .serializers import *
+from .models import *
 
 admin_group = settings.INTERVIEW_DB_AUTHZ_GROUPS['admin']
 front_end_group = settings.INTERVIEW_DB_AUTHZ_GROUPS['front-end']
 
-@method_decorator(group_required(front_end_group), name='dispatch')
-class IndexView(ListView):
-    template_name = 'index.html'
-    context_object_name = 'interview_list'
-    def get_queryset(self):
-        return Interview.objects.order_by('student')
 
+class PageView(TemplateView):
```

Comment:
I see you removed the group_required decorator, but it will probably still be needed, right? My understanding was there some AuthN still involved here. Doesn't have to be on this PR, but we should have a task tracking it wherever you're tracking tasks.